### PR TITLE
adds profile info drawer to message page

### PR DESCRIPTION
**Pull request by lspdrz**
[sc-1720]
DO NOT MERGE UNTIL https://github.com/Ameelio/connect-api-nest/pull/562 is merged and deployed

image

### CODE REVIEW

File: src/api/graphql.ts [hunk 1]
```diff
@@ -56,11 +58,24 @@ export type Connection = {
...
   inmate: Inmate;
   messages: MessageConnection;
+  relationship: ConnectionRelationship;
   restrictions: Array<ConnectionRestriction>;
   status: ConnectionStatus;
   visitor: Visitor;
 };
 
+export enum ConnectionRelationship {
+  Brother = "BROTHER",
```
Comment (wittejm) [inline src/api/graphql.ts:68]: Do these need to be gendered? We don't have gender on User profiles and these could all be Sibling, Parent, or Child (also Daughter is missing anyway :) )

Comment (lspdrz) [reply src/api/graphql.ts:68]: Ah, daughter is missing bc of a typo on my end. As for the gendered question, we'd need to talk to Liz

File: src/components/Inbox/InboxHeader.tsx [hunk 2]
```diff
@@ -1,21 +1,39 @@
...
+          <div className="flex items-center gap-2">
+            <VisitorAvatar visitor={connection.visitor} size="large" />
+            <span className="font-medium text-lg">
+              {getFullName(connection.visitor)}
+            </span>
+          </div>
+          <InfoCircleOutlined
+            className="cursor-pointer"
+            onClick={() => {setViewInfo(true)}}
+            style={{fontSize: '25px', color: '#8c8c8c'}}
```
Comment (wittejm) [inline src/components/Inbox/InboxHeader.tsx:28]: Can these be done with Tailwind?

**Review by wittejm — APPROVED** [review]
Still missing Daughter from the list. Otherwise LGTM.

File: src/components/Inbox/InboxHeader.tsx (hunk 2 above)
Comment (lspdrz) [reply src/components/Inbox/InboxHeader.tsx:28]: Unfortunately not, one of those antd things

Comment (lspdrz) [reply src/components/Inbox/InboxHeader.tsx:28]: The fontSize here is actually controlling the width of the icon hahaha

**Pull request merged**